Source/Private/Render/Windows/ServerWindow.cpp

```diff
@@ -133,20 +133,21 @@ void ServerWindow::Draw()
         ImGui::Text("Leave this game to start a new one.");
         ImGui::Separator();
         AutoPlayerSettings* aiSettings = Settings<AutoPlayerSettings>("AutoPlayers");
-        if (ImGui::Button("Start Game"))
+        if (ImGui::Button("START GAME"))
         {
             // These bots don't actually exist, it just tricks the server into thinking they do.
             aiSettings->ForceFillGameplayBotsTeam1 = 20;
             aiSettings->ForceFillGameplayBotsTeam2 = 19;
         }
         ImGui::Separator();
-        ImGui::Text("AI Settings:");
-        ImGui::SliderInt("AI Count", &aiSettings->ForcedServerAutoPlayerCount, -1, 64);
-        ImGui::Checkbox("Update AI", &aiSettings->UpdateAI);
-        ImGui::Checkbox("AI ignore players", &aiSettings->ServerPlayersIgnoreClientPlayers);
-        ImGui::Checkbox("Auto Balance Players", &Settings<WSGameSettings>("Whiteshark")->AutoBalanceTeamsOnNeutral);
+        ImGui::Text("AI SETTINGS");
+        ImGui::SliderInt("AI COUNT", &aiSettings->ForcedServerAutoPlayerCount, -1, 64);
+        ImGui::Checkbox("UPDATE AI", &aiSettings->UpdateAI);
+        ImGui::SameLine();
+        ImGui::Checkbox("AI IGNORE PLAYERS", &aiSettings->ServerPlayersIgnoreClientPlayers);
+        ImGui::Checkbox("AUTO BALANCE TEAMS", &Settings<WSGameSettings>("Whiteshark")->AutoBalanceTeamsOnNeutral);
         ImGui::Separator();
-        ImGui::Text("Player List:");
+        ImGui::Text("PLAYER LIST");
```

Comment:
If we're going to remove the colon I'd like to have this text be centered.

Reply:
I tried this but it looked kinda jank with everything else being left-aligned